### DISCUSSION TITLE
Docs per version ?

### DISCUSSION THREAD

**Discussion by Master-Y0da** [fastapi-users/fastapi-users#942] [Q&A] [answered]
I start using your library a long time a go (version 3.1.2) , so any documentation about this version for old loyal guys haha?

**Comment by frankie567** [comment, marked as answer]
Hmm, unfortunately, that's not something I can propose right now 🤔 

The only thing I can propose you is to directly read the markdown files from the 3.1.2 branch: https://github.com/fastapi-users/fastapi-users/tree/v3.1.2/docs

**Comment by Master-Y0da** [comment]
That will help, thank you!
